packages/golden_toolkit/test/font_loading_test.dart

```diff
@@ -38,6 +38,11 @@ Future<void> main() async {
             const Text('This should be italic',
                 style: TextStyle(
                     fontFamily: 'Roboto', fontStyle: FontStyle.italic)))
+        ..addScenarioContextWithContext('Italics are supported', (c) {
```

Comment:
I don't think this is the right test to add this behavior to.

It might be more appropriate in this file: https://github.com/eBay/flutter_glove_box/blob/master/packages/golden_toolkit/test/golden_builder_test.dart

the test should also actually demonstrate usage of the context value that is provided, in order to prove that it works. Perhaps accessing MediaQuery.of(context) or Theme.of(context) and using a property off of those?